woocommerce/woocommerce-ios
# [Mobille Payments] Remove inPersonPaymentGatewaySelection feature flag

Closes: #7593 


### Description

In "Make Cha-Ching not War", we added the ability for merchants in the USA to choose between WCPay and the Stripe extension for processing In-Person Payments. This was done behind a feature flag, [which has been `true` since release 9.6](https://github.com/woocommerce/woocommerce-ios/blob/9.6/Experiments/Experiments/DefaultFeatureFlagService.swift).

We are not likely to need to turn this off again, and our code will be simplified by removing the flag and the old way the app worked.

Previous to the "Make Cha-Ching Not War" project, merchants in this scenario were asked to disable or remove one of the plugins, via the DeactivateStripeView, PluginConflictAdminView, or PluginConflictShopManagerView. This removes those from the app as they are no longer required.

### Testing instructions


Test with stores in both Canada and the US, with both WCPay and Stripe installed and activated:

1. Go to the Payments tab, and tap continue setup. Complete plugin selection (if in USA) and the rest of onboarding. 
2. You should not see any onboarding screens telling you to disable a plugin.
3. Collect a payment. You should not see any onboarding errors, as long as you have previously selected a plugin.

---
